src/store/flow/block/set-contact-property.ts

```diff
@@ -0,0 +1,76 @@
+import {IRootState} from '@/store'
+import {findBlockOnActiveFlowWith as findBlock, IBlock, IContext, SetContactProperty} from '@floip/flow-runner'
+import {ActionTree, MutationTree} from 'vuex'
+import {IFlowsState} from '../index'
+
+export interface ISetContactPropertyIndex {
+  blockId: IBlock['uuid'],
+  key: SetContactProperty['property_key'],
+}
+
+export interface ISetContactPropertyIndexWithValue extends ISetContactPropertyIndex {
+  value: SetContactProperty['property_value'],
+}
+
+export const mutations: MutationTree<IFlowsState> = {
+  block_addContactProperty(state, {blockId, key, value}: ISetContactPropertyIndexWithValue) {
+    const block = findBlock(blockId, state as unknown as IContext)
+
+    // Ensure the block has a set_contact_property[]
```

Comment:
The code is self-explanatory (to me). I vote for removing these comments.
@safydy-r do you agree? Or should we keep them?

Reply:
I vote to remove it as well, thanks